lambdas/shared/t4_lambda_shared/preview.py

```diff
@@ -59,6 +60,27 @@ def decompress_stream(chunk_iterator, compression):
             break
 
 
+def extract_excel(file_, as_html=False):
```

Comment:
Might make sense to make it consistent with `extract_fcs()` and `extract_parquet()`:
```suggestion
def extract_excel(file_, as_html=True):
```

Reply:
done; for some weird reason did this on purpose thinking that default False was an anti-pattern